index.js

```diff
@@ -0,0 +1,50 @@
+import Bookstore from './modules/books.js';
+import formatAMPM from './modules/time.js';
+import SetupTabs from './modules/switch.js';
+
+const author = document.querySelector('#author');
+const title = document.querySelector('#title');
+const addBookForm = document.querySelector('#add-book');
+const list = document.querySelector('.list');
+const time = document.querySelector('.time');
+// /*eslint-disable*/
+// let luxon;
+// const { DateTime } = luxon;
+// /*eslint-disable*/
+
+setInterval(() => {
+  const today = new Date();
+  time.textContent = formatAMPM(today);
+}, 1000);
```

Comment:
- [ ] According to the requirement, you should use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library to display the current date below the navigation bar, the Module is imported like this: `import { DateTime } from "luxon";`

Reply:
good